### PR TITLE
[DC-357] Remove the test workaround for buckets

### PR DESCRIPTION
**Pull request by s-rubenstein**
This workaround no longer necessary with this change: https://github.com/DataBiosphere/terra-ui/pull/3013

### CODE REVIEW

**Comment by s-rubenstein** [comment]
I tested with `run-workflow-on-snapshot`.
For Dev: 7/100 failed, 2 unique errors
For this branch: 1/100 failed, 1 unique error

**Review by nawatts — APPROVED** [review]

File: integration-tests/tests/run-workflow.js [hunk 1]
```diff
@@ -17,8 +17,6 @@ const testRunWorkflowFn = _.flow(
 
   await createEntityInWorkspace(page, billingProject, workspaceName, testEntity)
   // Wait for bucket access to avoid sporadic failure when launching workflow.
```
Comment (cahrens) [inline integration-tests/tests/run-workflow.js]: Can you also remove this comment? Thanks for the cleanup!

**Review by cahrens — APPROVED** [review]
One very minor request, otherwise LGTM.

**Review by fboulnois — APPROVED** [review]
one suggestion: perhaps add why this workaround is no longer necessary in the PR description?

**Pull request merged**